framework/test/common/application.js

```diff
@@ -127,6 +127,12 @@ function __init(initScope, done) {
 		async.auto(
 			{
 				config(cb) {
+					__testContext.config.syncing.active = false;
+					__testContext.config.broadcasts.active = false;
```

Comment:
For the future reference - can you explain why exactly are you changing it?

Reply:
Before the dummy node we initiate for our integration & unit suites, wasn't able to connect to another node independent from the tests. Therefore tests were not affected. With the current PR, the dummy node is able to connect to another, fact that changes the expectations. For example the node is able to launch synchronisation process and receive extra unexpected blocks.